inc/passwords/namespace.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace HM\Platform\Security\Passwords;
+
+use function HM\Platform\get_config;
+use WP_Error;
+use ZxcvbnPhp\Zxcvbn;
+
+/**
+ * Bootstrap.
+ */
+function bootstrap() {
+	add_action( 'check_passwords', __NAMESPACE__ . '\\enforce_password_strength', 10, 3 );
```

Comment:
Actions should be sorted alphabetically.

Reply:
Hmm, I would generally disagree with this. Is there a particular benefit to sorting hooks alphabetically?

Reply:
Is this part of our guidelines?